src/server/passport/strategyUtils.js

```diff
@@ -0,0 +1,51 @@
+const responseManager = require('../routes/controllers/utilities/responseManager')
+const Utilities = require('../utilities')
+
+const STRATEGY_NAME = 'LOGIN_STRATEGY'
+const FIELDS_NAMES = {
+	USERNAME_FIELD: 'username',
+	PASSWORD_FIELD: 'password'
+}
+
+function strategyHandlers(req, res, next) {
+	const serializeHandler = (err) => {
+		if (err) return responseManager.createResponse(res, err)
+		
+		next()
+	}
+
+	const loginStrategyHandler = (err, user, info) => {
+		if (err)
+			return responseManager.createResponse(res, err)
+
+		if (user)	
+			req.logIn(user, serializeHandler)
+		else
+			return responseManager.createResponse(res, responseManager.MESSAGES.errors.UNAUTHORIZED)
+	}
+
+	return {
+		serializeHandler,
+		loginStrategyHandler
+	}		
+}
+
+function validateUserPassword(user, password, done) {
+	const hash = Utilities.createSaltedHash(user.salt, password)
+	if (hash === user.password) {
+		const error = null
+		done(error, user)
+		return true
```

Comment:
We don't test returned value, so I guess that we don't need it anymore.

Reply:
ok